lib/ProductOpener/Ingredients.pm

```diff
@@ -5943,12 +6037,13 @@ my %ingredients_categories_and_types = (
 			],
 			types => [
 				"arachide", "avocat", "carthame", "chanvre",
-				"coco", "colza", "coton", "illipe",
-				"karité", "lin", "mangue", "noisette",
-				"noix", "noyaux de mangue", "olive", "olive extra",
-				"olive vierge", "olive extra vierge", "olive vierge extra", "palme",
-				"palmiste", "pépins de raisin", "sal", "sésame",
-				"soja", "tournesol", "tournesol oléique",
+				"coco", "colza", "coprah", "coton",
+				"graines de colza", "illipe", "karité", "lin",
+				"mangue", "noisette", "noix", "noyaux de mangue",
+				"olive", "olive extra", "olive vierge", "olive extra vierge",
+				"olive vierge extra", "palme", "palmiste", "pépins de raisin",
+				"sal", "sésame", "soja", "tournesol",
+				"tournesol oléique",
```

Comment:
I though you where removing that !

Ain't there a way we could generate the list for the various languages from the english names ?

Or isn't it the case to make a small taxonomy for this kind of things ? (so that we can scale with languages).

Reply:
I am planning to remove that, but it will need a lot of testing and probably adjustments to check that the new way (introduced in this PR) supports those cases. Much more work is needed I think.